packages/sdk/odahuflow/sdk/clients/model.py

```diff
@@ -25,23 +25,23 @@
 import odahuflow.sdk.config
 from odahuflow.sdk.clients.deployment import ModelDeploymentClient
 from odahuflow.sdk.clients.route import ModelRouteClient
-from odahuflow.sdk.clients.api import Authenticator, IncorrectAuthorizationToken
+from odahuflow.sdk.clients.api import Authenticator, IncorrectAuthorizationToken, RemoteAPIClient
 from odahuflow.sdk.utils import ensure_function_succeed
 
 
 LOGGER = logging.getLogger(__name__)
 
 
-def calculate_url(base_url: str, model_route: str = None, model_deployment: str = None,
-                  url_prefix: str = None, mr_client: ModelRouteClient = None):
+def calculate_url(base_url: str, api_client : RemoteAPIClient, model_route: str = None,
```

Comment:
u can remove base_url from function arguments and extract it form api_client, but both solutions is more or less equal, let me know what do u prefer